www/gatsby-node.js

```diff
@@ -17,7 +17,7 @@ exports.onCreateWebpackConfig = function onCreateWebpackConfig({
 }) {
   actions.setWebpackConfig({
     devtool: stage.includes('develop')
-      ? 'cheap-inline-module-source-map'
+      ? 'inline-module-source-map'
```

Comment:
?

Reply:
the cheap one doesn't work well you can't set line breaks :/